plugin.service.gto/addon.xml

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.service.gto" name="German Telecast Offers" version="2.0.4" provider-name="Birger Jesch">
+    <requires>
+        <import addon="xbmc.python" version="2.20.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.service" library="starter.py" start="startup">
```

Comment:
As You know, calling a script or a plugin from outside is different. I've scripted and documented this like a script this handles (RunScript(scriptid,parameter)) instead of RunPlugin(pluginid?parameterhash). And, in fact i have more experience with writing scripts as with writing plugins...